internal/controller/postgrescluster/pgbackrest.go

```diff
@@ -2003,11 +2009,12 @@ func (r *Reconciler) reconcileManualBackup(ctx context.Context,
 func (r *Reconciler) reconcileReplicaCreateBackup(ctx context.Context,
 	postgresCluster *v1beta1.PostgresCluster, instances *observedInstances,
 	replicaCreateBackupJobs []*batchv1.Job,
-	serviceAccount *corev1.ServiceAccount, configHash, replicaCreateRepoName string) error {
+	serviceAccount *corev1.ServiceAccount, configHash string,
+	replicaCreateRepo v1beta1.PGBackRestRepo) error {
```

Comment:
I almost find it disturbing that you changed this func's signature, but because you changed `reconcileRepos` signature as well, you don't have to change where you invoke `reconcileReplicaCreateBackup` on  line 1322

Reply:
Ah, right. The variable storing the result of the former and passing to the latter was already called `replicaCreateRepo`. It used to be a string/name and now it is a "full" repo spec.